src/utilities/util_dump.c

```diff
@@ -652,18 +661,53 @@ dump_record(WT_CURSOR *cursor, bool reverse, bool json)
         infix = "\n";
         suffix = "\n";
     }
-    while ((ret = (reverse ? cursor->prev(cursor) : cursor->next(cursor))) == 0) {
-        if ((ret = cursor->get_key(cursor, &key)) != 0)
-            return (util_cerr(cursor, "get_key", ret));
-        if ((ret = cursor->get_value(cursor, &value)) != 0)
-            return (util_cerr(cursor, "get_value", ret));
-        if (fprintf(
-              fp, "%s%s%s%s%s%s", json && once ? "," : "", prefix, key, infix, value, suffix) < 0)
-            return (util_err(session, EIO, NULL));
-        once = true;
+
+    /* A specific is requested. */
+    if (key != NULL) {
+        current_key = key;
+        cursor->set_key(cursor, current_key);
+        if (search_near)
```

Comment:
Given that in a static (not live) system the `search_near` has (afaict) the same cost as `search`. Does it make sense to simply call `search_near`, possibly:
```suggestion
        ret = cursor->search_near(cursor, &exact);
        if (ret != 0 && !(ret == WT_NOTFOUND && search_near))
            return (util_cerr(cursor, search_near ? "search_near" : "search", ret));
```

Reply:
Yes, I agree! Thanks for the suggestion

Reply:
Updated in 3cf55340683e9a3b11a179d361d5c295ff59af58